src/compile.ts

```diff
@@ -66,9 +80,9 @@ const addMarkdownDescription = (obj: Record<string, any>) => {
       );
 
       // eslint-disable-next-line no-param-reassign
-      obj.markdownDescription = parsed;
-    } else if (isPlainObject(obj[key])) {
-      addMarkdownDescription(obj[key]);
+      obj.markdownDescription = addLink(pathname, parsed);
+    } else if (typeof obj[key] === 'object' && obj[key] !== null) {
```

Comment:
Ideally we would check for plain objects here, add another `else if` where we check for the array type and then iterate over each array element and check if it's a plain object and then call `addMarkdownDescription` for that element. That way we avoid using `for...in` for arrays, which is not recommended.

It seems to work for us though and the above approach just looks a lot less nice, so I just kept it like this. YOLO 😉